app/javascript/components/mentoring/inbox/conversation-list/Conversation.jsx

```diff
@@ -16,30 +17,54 @@ export function Conversation({
   updatedAt,
   url,
 }) {
+  const [referenceElement, setReferenceElement] = useState(null)
+  const [popperElement, setPopperElement] = useState(null)
+  const [showPopper, setShowPopper] = useState(null)
```

Comment:
Maybe we could initialize this state with a boolean `false` value? Makes it slightly more explicit that we're dealing with state containing a value that is interpreted as a boolean. Or is there something special about passing in `null`?

```suggestion
  const [showPopper, setShowPopper] = useState(false)
```